test/README.md

```diff
@@ -175,6 +175,13 @@ On non-Windows platforms, this currently returns true.
 
 Platform normalizes the `dd` command
 
+### expectsError(code[, type[, message]])
+* `code` [&lt;String>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Data_structures#String_type) expected error must have this value for its `code` property
+* `type` [&lt;constructor>](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/constructor) expected error must be an instance of `type`
```

Comment:
`constructor` is not actually a type...I think this is a `Function` and should be noted that it needs to be one that's subclassed by the `internal/errors` module? (Maybe a link to [the guide](https://github.com/nodejs/node/blob/master/doc/guides/using-internal-errors.md#api) here)